packages/embed/src/factories/create-widget/elements/build-widget.ts

```diff
@@ -1,6 +1,13 @@
-export const buildWidget = (iframe: HTMLIFrameElement): HTMLDivElement => {
+import { setElementSize } from '../../../utils'
+import { WidgetOptions } from '../widget-options'
+
+export const buildWidget = (iframe: HTMLIFrameElement, options: Omit<WidgetOptions, 'container'> = {}): HTMLElement => {
```

Comment:
It is not necessary to send whole `Omit<WidgetOptions, 'container'>` here just to get width and height. We can pass each one as separate argument (as you did for `buildSidetab`).